src/frame/content.rs

```diff
@@ -114,6 +116,19 @@ impl Content {
             Self::UniqueFileIdentifier(unique_file_identifier) => Comparable(vec![Cow::Borrowed(
                 unique_file_identifier.owner_identifier.as_bytes(),
             )]),
+            Self::InvolvedPeopleList(involved_people_list) => Comparable(
```

Comment:
The ID3v2.3 spec states that an Involved People List may only occur once in a tag. I think it would be okay to assume that this property extends to v2.4.

This `unique()` function is used to determine whether there are duplicate tags based on their unique properties. For a tag that may only exist once, it should return `Same` here.

Reply:
In ID3v2.4 there is also the `TMCL` frame, which is also such a list:

```
  TMCL
   The 'Musician credits list' is intended as a mapping between
   instruments and the musician that played it. Every odd field is an
   instrument and every even is an artist or a comma delimited list of
   artists.

  TIPL
   The 'Involved people list' is very similar to the musician credits
   list, but maps between functions, like producer, and names.
```

Should this still return `Same`?

Reply:
Yes, I think so.

Uniqueness is based on the frame ID and its content as specified by the spec. So returning `Same` for TMCL, TIPL, etc will still permit all of those frames to coexist in a tag

Reply:
84a00a318186df0bb78187114f994d52b9f29085